main/middleware.py

```diff
@@ -7,6 +8,8 @@ class CachelessAPIMiddleware(MiddlewareMixin):
 
     def process_response(self, request, response):
         """ Add a Cache-Control header to an API response """
-        if request.path.startswith("/api/"):
+        if request.path.startswith(settings.CACHEABLE_ENDPOINTS):
+            response["Cache-Control"] = "max-age=3600, public"
```

Comment:
Should the max-age be configurable via .env variables too?

Reply:
Rather than creating a variable for max-age, I have introduced `CACHEABLE_ENDPOINTS_CACHE_VALUE` so the whole cache-control value can be changed. Kindly have a look now.